### ISSUE TITLE
bootstrap.sh overrides freshclam.conf Checks setting

### ISSUE THREAD

**Issue by DavidJFowler** [mko-x/docker-clamav#106]
**Describe the bug**
[bootstrap.sh](https://github.com/mko-x/docker-clamav/blob/master/alpine/edge/bootstrap.sh) initialises freshclam with this command:
```sh
echo "[bootstrap] Schedule freshclam DB updater."
/usr/bin/freshclam -d -c 6
```
The `-c 6` option overrides any value set for `Checks` in `freshclam.conf`

**To Reproduce**
Configure a custom freshclam.conf with `Checks` set to a value other than 6.

**Expected behavior**
freshclam should be scheduled without the `-d` command line option and use the value of `Checks` set in freshclam.conf or the default.

**Image (please complete the following information):**
 - Tag: alpine:edge

**Comment by mko-x** [comment]
Seems to make sense to me, could you provide a PR? I think we may have the problem in the other images as well.

**Comment by DavidJFowler** [comment]
@mko-x Created PR https://github.com/mko-x/docker-clamav/pull/110

**Comment by mko-x** [comment]
Fixed by #110 - Thank you very much @DavidJFowler
